device-types/Fortinet/FG-100D.yaml

```diff
@@ -0,0 +1,66 @@
+---
+manufacturer: Fortinet
+model: FortiGate 100D
+slug: fg-100d
```

Comment:
Please append `fortinet-` to the slug

Reply:
e048a3d52d2594bb5dc35b44384775c258ad03fa